### ISSUE TITLE
layout of shopping items on scale and shelf

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/unit-rates#167]
I'm creating this issue primarily to track decisions that we make.  

In the original implementation (e.g. [1.0.0-dev.19](http://www.colorado.edu/physics/phet/dev/html/unit-rates/1.0.0-dev.19/unit-rates_en.html)), shopping items were arranged in front/back rows on the shelf:

![screenshot_165](https://cloud.githubusercontent.com/assets/3046552/23535146/e420cc18-ff79-11e6-8307-b5931d102daf.png)

... while items on the scale were stacked in two rows:

![screenshot_166](https://cloud.githubusercontent.com/assets/3046552/23535162/fd65c8cc-ff79-11e6-90de-4aa06b53c1ba.png)

Since there doesn't seem to be any pedagogical or usability reason for this difference, it would be better to have a similar layout (and therefore single implementation) on both scale and shelf.  

For fruit, the shelf can contain both bags and items. Bags are in a their own row behind items. So stacking items on the shelf could make it a little tricky to grab a bag that is behind the items. 

Stacking also looks a bit unnatural, for example:

![screenshot_167](https://cloud.githubusercontent.com/assets/3046552/23535281/a9867c82-ff7a-11e6-99d9-070404ec95ac.png)

The front/back layout makes it a little more difficult to count items on the scale.  And more difficult to grab items.  But theses issues could easily be addressed by increasing space between the items, and changing the viewing perspective (effectively tilting the scale and shelf a bit more).

**Comment by pixelzoom** [comment]
In [1.0.0-dev.64](http://www.colorado.edu/physics/phet/dev/html/unit-rates/1.0.0-dev.64/unit-rates_en.html?showAnswers&randomEnabled=false), I implemented the "front/back layout" for both the shelf and scale. 

Caveats: This implementation is incomplete. Fruit bags do not open yet, so the shelf is initially populated with some items. If you put bags and individual items on the scale, keep in mind that this will never happen for fruit.  That said…

I replaced the scale PNG file with a programmatic implementation. (I had done this awhile back for the shelf.) This allowed me to easily change the viewing perspective, effectively tilting both the scale and the shelf, and making it easier to see (and count) individual items.  The scale doesn’t look identical to the PNG file, but does include gradient shading, and I added a shadow under the top platter of the scale.  Both the scale and shelf appear to be deeper due to the new viewing perspective angle.  Here's a sample screenshot:

![screenshot_168](https://cloud.githubusercontent.com/assets/3046552/23535469/c20fb0d8-ff7b-11e6-92f5-2f509288e566.png)

Overall, I like this approach. It’s easy to count items on the scale.  Bags and items coexist nicely on the shelf.  Bags and items have plenty of space between them, making them easy to grab.  And it certainly looks more natural than the stacking approach.  Compare this layout of Pears to the old one shown above:

![screenshot_169](https://cloud.githubusercontent.com/assets/3046552/23535514/f96ad3b4-ff7b-11e6-9085-c3350fb19d4d.png)

@arouinfar your feedback?
